res/css/views/typography/_Heading.pcss

```diff
@@ -33,6 +33,7 @@ limitations under the License.
 .mx_Heading_h3 {
     font: var(--cpd-font-heading-md-semibold);
     font-weight: var(--cpd-font-weight-semibold);
+    letter-spacing: var(--cpd-font-letter-spacing-heading-lg);
```

Comment:
Why `-lg` and not `-md`?

Reply:
So this was the letter spacing used for `h2`s so it's what makes them match what they were before. I'm also wondering why they're styled as h3s though - this might be what makes it odd.